paradex_py/api/ws_client.py

```diff
@@ -190,7 +195,7 @@ def _check_subscribed_channel(self, message: dict) -> None:
 
     async def _read_messages(self):
         while True:
-            if self.ws and self.ws.open:
+            if self.ws and self.ws.state == State.OPEN:
```

Comment:
https://websockets.readthedocs.io/en/stable/howto/upgrade.html#open-and-closed should not be used?